### ISSUE TITLE
Voluteer Signup does not set VMS Permission level on SF contact

### ISSUE THREAD

**Issue by WaBirder57** [washingtontrails/vms#43]
When a Volunteer registers and creates a my backpack account the SF contact is also created.  This should also set the Contact.VMS Permission Level to Volunteer (if it is a new contact).  If the contact already exists and there is no value in this field it should also be set to Volunteer, otherwise don't change it.

**Comment by davisagli** [comment]
If the contact's VMS Permission Level is empty, VMS assumes the contact has a permission level of Volunteer. So this seems pretty low priority, right?

**Comment by WaBirder57** [comment]
Yes, are you saying the VMS equates a permission level of "Volunteer" to be the same as when it is empty?

**Comment by davisagli** [comment]
Right. When you log into VMS it fetches the contact info from Salesforce, and if the permission level is empty it uses Volunteer.
